draft-ietf-quic-transport.md

```diff
@@ -1834,7 +1834,12 @@ new connection IDs using the NEW_CONNECTION_ID frame.
 An endpoint might need to send packets on multiple networks without receiving
 any response from its peer.  To ensure that the endpoint is not linkable across
 each of these changes, a new connection ID is needed for each network.  To
-support this, multiple NEW_CONNECTION_ID messages are needed.
+support this, multiple NEW_CONNECTION_ID messages are needed.  Each
+NEW_CONNECTION_ID is marked with a sequence number.  Connection IDs MUST be used
+on only one local address, and each local address MUST advance to a connection
```

Comment:
Perhaps
... on only one local address.  An endpoint MUST use a connection ID with a higher sequence number if it changes to using a different local address or network interface.  Once an endpoint has used a connection ID for packets sent from a given local address, they MUST NOT use a connection ID with a lower sequence number with that address.

Reply:
Or, create a new section on connection ID use along the lines I describe below and cite that instead.